src/Avalonia.Controls/Embedding/Offscreen/OffscreenTopLevelImpl.cs

```diff
@@ -59,7 +59,7 @@ public double Scaling
         public virtual void SetCursor(IPlatformHandle cursor)
         {
         }
-
```

Comment:
Since you have added and then removed code, please rebase and squash your PR. This will keep the history of unaffected files clean. 

Also - https://github.com/AvaloniaUI/Avalonia/pull/1383/files#diff-d0f7130a78a934890ba6e5ea90c7b14bR37

Reply:
Sorry, this should be cleaned up now.